engine/render/src/render/font_renderer_private.h

```diff
@@ -120,20 +91,24 @@ namespace dmRender
         dmGraphics::TextureFilter m_MinFilter;
         dmGraphics::TextureFilter m_MagFilter;
 
-        uint32_t                m_CacheWidth;           // In texels
-        uint32_t                m_CacheHeight;          // In texels
-        uint32_t                m_CacheCellWidth;       // In texels
-        uint32_t                m_CacheCellHeight;      // In texels
-        uint32_t                m_CacheCellMaxAscent;   // In texels
-        uint32_t                m_CacheColumns;         // Number of cells in horizontal direction
-        uint32_t                m_CacheRows;            // Number of cells in horizontal direction
-        uint32_t                m_CacheCellCount;       // Number of cells in total
+        uint16_t                m_CacheMaxWidth;        // In texels
```

Comment:
Suggest to use here and below `uint16_t : 15;` because we can handle texture of maximum size 32k. So we don't need 65k maximum value. And we can use free bits for some of uint8_t value. Small but still optimization)